PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobDebuggerRunner.java

```diff
@@ -120,23 +120,13 @@ public GenericDebuggerRunnerSettings createConfigurationData(ConfigurationInfoPr
         return null;
     }
 
-    private void checkSubmissionParameter(SparkSubmissionParameter parameter) throws ExecutionException {
-        if (StringUtils.isEmpty(parameter.getArtifactName())) {
-            throw new ExecutionException("The artifact to submit is not selected, please config it at Run/Debug configuration -> Remotely Run in Cluster.");
-        }
-
-        if (StringUtils.isEmpty(parameter.getClusterName())) {
-            throw new ExecutionException("The HDInsight cluster to submit is not selected, please config it at Run/Debug configuration -> Remotely Run in Cluster.");
-        }
-    }
-
     @Override
     protected void execute(@NotNull ExecutionEnvironment environment, @Nullable Callback callback, @NotNull RunProfileState state) throws ExecutionException {
         SparkBatchJobSubmissionState submissionState = (SparkBatchJobSubmissionState) state;
         SparkSubmitModel submitModel = submissionState.getSubmitModel();
         Project project = submitModel.getProject();
 
-        checkSubmissionParameter(submitModel.getSubmissionParameter());
+        submissionState.checkSubmissionParameter();
```

Comment:
>submissionState.checkSubmissionParameter(); [](start = 8, length = 43)

Should we also move this to SparkBatchJobSubmissionState?

Reply:
I have to put it here for remote debugging since they are different code path currently. It might be changed when the remote debugging refectory is finished.